packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -633,6 +635,17 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
                 m.isVisible = !selectedMesh || m == selectedMesh;
             });
     }
+
+    protected _unhoverMeshOnTouchUp(pointerId: number, selectedMesh: AbstractMesh) {
+        const engine = this.gizmoLayer.originalScene.getEngine();
+        const isMobileSafari = engine._badOS || (engine._badDesktopOS && typeof document === "object" && "ontouchend" in document);

```

Comment:
`document` may not exist when running in BabylonNative. I would add a test for that.

Reply:
mobile safari hack is no longer needed as of 16309d8